Sources/FaceLiveness/AV/LivenessCaptureSession.swift

```diff
@@ -121,6 +121,7 @@ class LivenessCaptureSession {
             .filter(\.isVideoOrientationSupported)
             .forEach {
                 $0.videoOrientation = .portrait
+                $0.isVideoMirrored = true
```

Comment:
is there a way to test this?

Reply:
I had to test this manually since it requires a live camera session